public/css/madras-theme.css

```
@@ -664,6 +664,86 @@ body::after {
+  padding-bottom: 1.75rem;
+  border-bottom: 1px solid var(--sandalwood);
+}
+
+.shop-item:last-child {
+  margin-bottom: 0;
+  padding-bottom: 0;
+  border-bottom: none;
+}
+
+.shop-item h3 {
+  font-family: 'Spectral', serif;
```

Stylelint (17.14.1)

[error] Expected no quotes around "Spectral" (font-family-name-quotes)

(font-family-name-quotes)